content/blog/aws-gke-token-exchange/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'OAuth Token Exchange: AWS → GCP'
+date: '2021-12-03T11:16:00.0Z'
+description: How to exchange short lived tokens between AWS and GCP
+tags: ['AWS', 'GCP', 'authentication']
+author:
+  name: Nicolas Arnold
+  avatar: '../../assets/team/nic.jpeg'
+---
+
+When working with multiple cloud providers, it can often become difficult to manage authentication. Even more so with inter communication. In this blog post, I will talk about my experience with negotiating AWS identity tokens for GCP OAuth tokens.
+
+Normally, when trying to gain access to another AWS account, we use cross federation. With this cross federation, we authorize access to certain AWS principals (roles, users etc). This is done by assuming a "role". This "role" is exclusively controlled by the owner's account. The account owners can determine exactly what access the external account has. With this, we are able to provide a secure way for two (or more) AWS accounts to communicate with each other.
```

Comment:
It shouldn't be possible to assume users so I think that's likely an incorrect statement here.